sbi/inference/trainers/base.py

```diff
@@ -306,6 +320,239 @@ def train(
     ) -> NeuralPosterior:
         raise NotImplementedError
 
+    @abstractmethod
+    def _get_potential_function(
+        self,
+        prior: Distribution,
+        estimator: Union[RatioEstimator, ConditionalEstimator],
+    ) -> Tuple[BasePotential, TorchTransform]:
+        """Subclass-specific potential creation"""
+        pass
```

Comment:
I think here the convention is `...` and maybe then code coverage will ignore this.